resources/views/vendor/cookie-consent/dialog.blade.php

```diff
@@ -42,10 +43,16 @@
             <div class="flex items-center justify-between flex-wrap">
                 <div class="w-0 flex-1 items-center hidden md:inline">
                     <p class="ml-3 text-primary-950 text-sm font-medium cookie-consent__message">
-                        @if(! empty(app(PortalSettings::class)->gdpr_banner_text))
-                            {!! str(app(PortalSettings::class)->gdpr_banner_text)->sanitizeHtml() !!}
+                        @if(FeatureFlag::GDPRNewBannerText->active())
+                            {!! tiptap_converter()->asHtml(app(PortalSettings::class)->cookie_gdpr_banner_text) !!}
```

Comment:
Do we need to consider sanitizing this HTML? I forget if the `tiptap_converter` `asHtml` method does that for you.